frontend/src/metabase/embedding/embedding-hub/components/StepperWithCards/StepperWithCards.tsx

```diff
@@ -89,109 +93,118 @@ export const StepperWithCards = ({ steps }: { steps: StepperStep[] }) => {
+                                          c="var(--mb-color-success-darker)"
+                                        >
+                                          {t`Done`}
+                                        </Text>
+                                      </Group>
+                                    ))
+                                    .with({ locked: true }, () => (
+                                      <Group gap="xs">
+                                        <Icon
+                                          name="lock"
+                                          c="var(--mb-color-text-secondary)"
+                                          size={12}
+                                        />
+
+                                        <Text
+                                          c="var(--mb-color-text-secondary)"
+                                          fz={12}
+                                        >
+                                          {t`Complete the other steps to unlock`}
+                                        </Text>
+                                      </Group>
+                                    ))
+                                    .with({ optional: true }, () => (
                                       <Text
+                                        size="sm"
                                         c="var(--mb-color-text-secondary)"
-                                        fz={12}
                                       >
-                                        {t`Complete the other steps to unlock`}
+                                        {t`Optional`}
                                       </Text>
-                                    </Group>
-                                  ))
-                                  .with({ optional: true }, () => (
-                                    <Text
-                                      size="sm"
-                                      c="var(--mb-color-text-secondary)"
-                                    >
-                                      {t`Optional`}
-                                    </Text>
-                                  ))
-                                  .otherwise(() => null)}
-                              </Flex>
-                            )}
-                          </Stack>
-                        </Card>
-                      </CardAction>
-                    </Grid.Col>
-                  );
-                })}
-              </Grid>
+                                    ))
+                                    .otherwise(() => null)}
+                                </Flex>
+                              )}
+                            </Stack>
+                          </Card>
+                        </CardAction>
+                      </Grid.Col>
+                    );
+                  })}
+                </Grid>
+
+                {step.alert && (
+                  <StepAlert
```

Comment:
This renders the alert box below the step card, for the "Get embed snippet" step.

<img width="1382" height="576" alt="CleanShot 2568-12-18 at 21 52 22@2x" src="https://github.com/user-attachments/assets/431aa604-1c7f-4353-a2d0-830a1e33afbf" />